src/components/Explore/FavoriteButton.tsx

```diff
@@ -1,37 +1,45 @@
 import useTheme from 'hooks/useTheme'
 import { useAtom } from 'jotai'
-import { showFavoritesAtom } from 'pages/Explore/index'
+import { darken } from 'polished'
 import { Heart } from 'react-feather'
 import styled from 'styled-components/macro'
 
+import { showFavoritesAtom } from './TokenTable'
+
 const FavoriteButtonContent = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
   gap: 8px;
 `
-const StyledFavoriteButton = styled.button`
+const StyledFavoriteButton = styled.button<{ active: boolean }>`
   padding: 0px 16px;
   border-radius: 12px;
   background-color: ${({ theme }) => theme.bg0};
-  border: 1px solid ${({ theme }) => theme.bg0};
-  color: ${({ theme }) => theme.text1};
+  border: 1px solid ${({ theme, active }) => (active ? '#869EFF' : theme.bg1)};
+  color: ${({ theme, active }) => (active ? '#869EFF' : theme.text1)};
   font-size: 16px;
   cursor: pointer;
 
-  :hover {
+  &:hover,
+  &:focus {
     outline: none;
-    border: 1px solid ${({ theme }) => theme.bg3};
+    color: ${({ theme, active }) => (active ? '#869EFF' : darken(0.08, theme.text1))};
   }
 `
 
 export default function FavoriteButton() {
   const theme = useTheme()
   const [showFavorites, setShowFavorites] = useAtom(showFavoritesAtom)
   return (
-    <StyledFavoriteButton onClick={() => setShowFavorites(!showFavorites)}>
+    <StyledFavoriteButton onClick={() => setShowFavorites(!showFavorites)} active={showFavorites}>
       <FavoriteButtonContent>
-        <Heart size={17} color={theme.text1} fill={theme.text1} /> Favorites
+        <Heart
+          size={17}
+          color={showFavorites ? '#869EFF' : theme.text1}
```

Comment:
sometimes you might hear people say DRY (https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) - which basically just means, if you have the same block of code, write it once instead of twice. so you could make this a variable above